pipelines/azure-tests.yaml

```diff
@@ -22,15 +44,34 @@ jobs:
         displayName: Set python version
         inputs:
           versionSpec: "$(python.version)"
+      - checkout: self
+        fetchDepth: 1
+        path: $(self_location)
+      - checkout: unittest-samples
+        fetchDepth: 1
+        path: $(samples_location)
       - script: |
+          [ ! -d "$(pwd)/tests" ] && echo "No tests found" && exit
           sudo apt-get update
-          sudo apt-get install -y qemu-utils libfuzzy-dev libfuzzy2
```

Comment:
I think you lost the installation of qemu-utils in the process?

Reply:
qemu-utils is not required